install.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/sh
+set +x
+#------------------------------------
+# Run global-workflow installation
+#
+# 1 - run checkout
+# 2 - run build
+# 3 - run linking
+#------------------------------------
+
+while getopts ":m:t:CBL" option;
+do
+ case $option in
+  m)
+   echo received -m with $OPTARG
+   mode=$OPTARG
+   ;;
+  t)
+   echo received -t with $OPTARG
+   TOPDIR=$OPTARG
+   ;;
+  C)
+   echo received -C
+   CHECKOUT="YES"
+   ;;
+  B)
+   echo received -B
+   BUILD="YES"
+   ;;
+  L)
+   echo received -L
+   LINK="YES"
+   ;;
+  :)
+   echo "option -$OPTARG needs an argument"
+   ;;
+  *)
+   echo "invalid option -$OPTARG, exiting..."
+   exit
+   ;;
+ esac
+done
+
+rc=0 # Set return code
+ME_VER=manic-v1.1.8 # Set manage_externals version
+
+mode=${mode:-'dev'} # "dev" - not production installs; "prod" - NCO production installation
+TOPDIR=${TOPDIR:-`pwd`}
+CHECKOUT=${CHECKOUT:-"NO"}
+BUILD=${BUILD:-"NO"}
+LINK=${LINK:-"NO"}
+# If no feature flags set, set all to YES
+if [ $CHECKOUT = "NO" -a $BUILD = "NO" -a $LINK = "NO" ]; then
+  CHECKOUT="YES"; BUILD="YES"; LINK="YES"
+fi
+
+#------------------------------------
+# GET MACHINE
+#------------------------------------
+target=""
+source $TOPDIR/sorc/machine-setup.sh > /dev/null 2>&1
+
+#------------------------------------
+# RUN CHECKOUT
+#------------------------------------
+if [ $CHECKOUT = "YES" ]; then
+
+  # Set manage_externals path
+  if [ $target = "wcoss_dell_p3" ]; then
+    ME_INSTALL=/gpfs/dell2/emc/modeling/noscrub/emc.glopara/git/manage_externals/$ME_VER
```

Comment:
Is mange_externals a software/tool can be found in any platforms ?  Is it available on non-NOAA computers, for example, S4, Cheyenne, Stampede etc ?

Reply:
I have installs of the latest tag on the WCOSSes and Hera but want to get it set up as a module. It's not available otherwise.

Reply:
I am going to remove this new script from this branch and bring it in via another one. I'd like to exercise it more before making it available. Will move to my fork for now.